Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -26,7 +26,8 @@
     sprintModifier: 0.80
   - type: HeldSpeedModifier
   - type: ToggleableClothing
-    clothingPrototype: ClothingHeadHelmetHardsuitBasic
+    clothingPrototypes: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)